public/docs/ts/latest/glossary.jade

```diff
@@ -520,36 +507,35 @@ a#N
     Events stream *out* of this property to the receiver identified
     in the template expression to the right of the equal sign.
 
-    See the [Template Syntax](!{docsLatest}/guide/template-syntax.html#inputs-outputs) chapter.
+    See the [Template Syntax](!{docsLatest}/guide/template-syntax.html#inputs-outputs) page.
 
 .l-main-section#P
 
 :marked
   ## PascalCase
 .l-sub-section
   :marked
-    The practice of writing compound words or phrases such that each word or abbreviation begins with a capital letter.
-    Class names are typically spelled in PascalCase. Examples include: `Person` and `HeroDetailComponent`.
+    The practice of writing individual words, compound words, or phrases such that each word or abbreviation begins with a capital letter. Class names are typically spelled in PascalCase. Examples include: `Person` and `HeroDetailComponent`.
 
-    This form is also known as **upper camel case**, to distinguish it from **lower camel case** which we simply call [camelCase](#camelcase).
+    This form is also known as **upper camel case** to distinguish it from **lower camel case** which we simply call [camelCase](#camelcase).
```

Comment:
Done. Did a search for this and replaced another occurrence elsewhere in the same file, too.